sample-notificator-domain/src/main/groovy/life/qbic/business/subscription/Subscriber.groovy

```diff
@@ -0,0 +1,34 @@
+package life.qbic.business.subscription
+
+import life.qbic.datamodel.samples.Status
+
+/**
+ * <h1>A person that subscribed to the notification table</h1>
+ *
+ * <p>A subscriber is described by his full name and an email address</p>
+ *
+ * @since 1.0.0
+ *
+*/
+class Subscriber {
+    final String firstName
+    final String lastName
+    final String email
+    /**
+     * Mapping sample codes to its new sample status
+     */
+    final Map<String, Status> subscriptions
```

Comment:
we do not subscribe to single samples but to projects

Reply:
![image](https://user-images.githubusercontent.com/11536497/128994106-7a1d1c0b-5ea1-45b3-8829-84b7425b2ebc.png)

Reply:
true, but we need the information about the sample. And we can infer the project information from a sample.